multinet/multinet.py

```diff
@@ -107,4 +107,7 @@ def create_graph(workspace, graph):
         error = {"error": "undefined_tables", "detail": list(invalid_from | invalid_to)}
         return (error, "400 Edge Table Validation Failed")
 
-    return graph if db.create_graph(workspace, graph, node_tables, edge_table) else None
+    if db.create_graph(workspace, graph, node_tables, edge_table):
+        return graph
+    else:
+        return (f"Graph {graph} already exists", 400)
```

Comment:
I think it would be better to return `graph` (to identify the problem), and use `"400 Graph Already Exists"` for the status code. (We could also consider using [409](https://www.restapitutorial.com/httpstatuscodes.html) for the error code, which might be good for helping the client disambiguate what's going on.)